aep/general/0124/aep.md.j2

```diff
@@ -1,5 +1,86 @@
 # Resource association
 
-**Note:** This AEP has not yet been adopted. See
-[this GitHub issue](https://github.com/aep-dev/aep.dev/issues/57) for more
-information.
+APIs sometimes have resource hierarchies that can not be cleanly expressed in
+the usual tree structure. For example, a resource may have a many-to-one
+relationship with two other resource types instead of just one. Alternatively,
+a resource may have a many-to-many relationship with another resource type.
+
+## Guidance
+
+A resource **must** have at most one canonical parent, and `List` requests
+**must not** require two distinct "parents" to work.
+
+### Multiple many-to-one associations
+
+If a resource has a many-to-one relationship with multiple resource types, it
+**must** choose at most one of them to be the canonical parent. The resource
+**may** be associated with other resources through other fields on the
+resource.
+
+{% tab proto %}
+
+{% sample 'multiple_many_to_one.proto', 'message Book' %}
+
+{% tab oas %}
+
+{% sample 'multiple_many_to_one.oas.yaml', 'Book:' %}
```

Comment:
```suggestion
{% sample 'multiple_many_to_one.oas.yaml', 'Book' %}
```

Without this change, the rendering looks weird. I'd guess it's because it's working off of the key string:


![2024-03-15-211817_728x307_scrot](https://github.com/aep-dev/aep.dev/assets/391240/42531d92-db77-45d5-bc19-0cc48d4409e4)

the rendering on this looks very strange.

Reply:
How does the sample block work? My initial guess is that the Jinja2 tag looks for the line ('Book') and then prints everything that's nested underneath it (defined as every line being 1+ spaces more nested than that original line).

I think your changes break my guess though.